.github/workflows/assets.yml

```diff
@@ -75,6 +75,13 @@ jobs:
              mv assets/kernel assets/kernel.gz
              gzip -d assets/kernel.gz
           fi
+      - name: Build SBOM and collected sources
+        run: |
+            git clone ${{ github.event.repository.html_url }} .
```

Comment:
You can replace this with a checkout action. See for example [here](https://github.com/lf-edge/eve/blob/master/.github/workflows/build.yml#L41-L43). If you do, it should be early on.

Reply:
It also should move you into the correct directory, so you will be in `lf-edge/eve` dir, where the `Makefile` and everything else you need is available.

Reply:
will use the checkout action.

Reply:
used the checkout option